src/protocol/properties.rs

```diff
@@ -222,10 +222,13 @@ impl Default for CompletionOptions {
         CompletionOptions {
             resolve_provider: Some(true),
             trigger_characters: Some(vec![
```

Comment:
I am concerned that this will cause erroneous completions and lead to complex completion logic moving forward, we saw a similar issue when the UI add a larger set of trigger characters.